database/evaluation_db.sql

```diff
@@ -166,6 +165,37 @@ INSERT INTO `faculty_list` (`id`, `school_id`, `firstname`, `lastname`, `email`,
 
 -- --------------------------------------------------------
 
+--
+-- Table structure for table `login`
+--
+
+CREATE TABLE `login` (
+  `school_id` int(11) NOT NULL,
+  `firstname` varchar(255) NOT NULL,
+  `lastname` varchar(255) NOT NULL,
+  `email` varchar(255) NOT NULL,
+  `password` varchar(255) NOT NULL,
+  `class_id` int(11) NOT NULL,
+  `avatar` blob NOT NULL,
+  `date_created` datetime NOT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci;
+
+--
+-- Dumping data for table `login`
+--
+
+INSERT INTO `login` (`school_id`, `firstname`, `lastname`, `email`, `password`, `class_id`, `avatar`, `date_created`) VALUES
+(2147483647, 'First Name', 'Last Name', 'Email', '$2y$10$V92q2fOjeVF2w3nytZW.0ujYaXRIkgGjvgxM4YeORjH.DYoeghPgG', 0, 0x6e6f2d696d6167652d617661696c61626c652e706e67, '2024-05-11 18:07:06'),
```

Comment:
Is this SQL exported from the local db setup? 
I suggest you can remove the repetitive records here

Reply:
I have removed the SQL which was exported from my local db setup